cmake/projects/OpenSSL/externals/nasm_path_configure.cmake.in

```diff
@@ -0,0 +1,24 @@
+cmake_minimum_required(VERSION 3.2)
```

Comment:
I think name `configure.cmake.in` is more appropriate. Also same with Qt you can put it into `ep-stages` folder: `OpenSSL/ep-stages/configure.cmake.in`.